app/models/topic.rb

```diff
@@ -1,4 +1,6 @@
 class Topic < ApplicationRecord
+  include SoftDeletion
```

Comment:
Topics are the old forum, only available on the old theme and I believe are no longer even able to be created. So we can leave them out of the equation.

Reply:
My only reason to add it was because we efficiently destroy them on `before_destroy` in [`user.rb`](https://github.com/sudara/alonetone/blob/master/app/models/user.rb#L229)

What about `features`? Wasn't sure about them either.

Reply:
Oooh yeah, features are also old data, only kept for data posterity, I need to suck it up and delete em!